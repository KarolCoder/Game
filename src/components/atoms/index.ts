export {Spacer} from './Spacer';
export {Card} from './Card';
