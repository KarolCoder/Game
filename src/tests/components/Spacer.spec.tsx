import {Spacer} from '@/components';
import {render} from '@testing-library/react-native';
import React from 'react';

import renderer from 'react-test-renderer';

it('spacer renders correctly', async () => {
  const component = <Spacer width={12} height={12} />;
  const result = renderer.create(component).toJSON();
  const {UNSAFE_getByProps} = render(component);
  expect(result).toMatchSnapshot();
  UNSAFE_getByProps({width: 12});
  UNSAFE_getByProps({height: 12});
});
