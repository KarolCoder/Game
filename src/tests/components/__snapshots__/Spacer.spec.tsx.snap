// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`spacer renders correctly 1`] = `
<View
  height={12}
  style={
    Array [
      Object {
        "height": 12,
        "width": 12,
      },
    ]
  }
  width={12}
/>
`;
